packages/block-editor/src/components/list-view/index.js

```diff
@@ -257,12 +260,20 @@ function ListViewComponent(
 		return null;
 	}
 
+	const describedById =
+		description && `block-editor-list-view-description-${ instanceId }`;
+
 	return (
 		<AsyncModeProvider value={ true }>
 			<ListViewDropIndicator
 				listViewRef={ elementRef }
 				blockDropTarget={ blockDropTarget }
 			/>
+			{ description && (
+				<VisuallyHidden id={ describedById }>
```

Comment:
@afercia As we found in another PR, there is actually a current bug in NVDA that does not allow `aria-describedby` to work properly if an element is hidden. I think visually hidden is okay as long as it is placed away from the users main interaction pattern. This is where `aria-description` could make a huge difference if browsers would just gain support for it.

Reply:
> there is actually a current bug in NVDA that does not allow `aria-describedby` to work properly if an element is hidden. I think visually hidden is okay as long as it is placed away from the users main interaction pattern.

@alexstine ok, even though I'm not sure we should address screen readers bugs, I'm ok with VisuallyHidden _for now_ untile the NVDA bug gets solved. Are you tracking the NVDA bug? It would be useful to add a link to the  bug tracker her for reference..

> This is where `aria-description` could make a huge difference if browsers would just gain support for it.

I agree and `aria-description` would also be more intuitive and easy to use for developers. However, WordPress supports the last 2 major browser versions. Even when it will be supported by the last 2 major browser versions, we will need to keep into considerations old screen reader versions may not behave correctly. Technically, `aria-description` should be exposed as accessible description so I wouldn't expect problems. However, when we will implement it, we should text also with old screen readers. Worth reminding that some commercial screen readers are costly and users tend to stick with old versions for years.